index.html

```diff
@@ -153,14 +168,35 @@ <h2>Schedule (to be defined)</h2>
 
 
 	<h2>Venue</h2>
-	The venue for this event is the Akasha Hub Barcelona, located at<br>
-		Verneda Street, number 19, Warehouse 1, 08018 Barcelona (Spain).
+	The venue for this event is the <b>Akasha Hub Barcelona.</b><br>
```

Comment:
How would you see to have a link to their website?

```suggestion
	The venue for this event is the <b><a href="https://akasha.barcelona" target="_blank">Akasha Hub Barcelona.</a></b><br>
```